<launch>
    <arg name="Mode" default="1" /> <!-- 0:ENU, 1:LLH, 2:ECEF -->
    <arg name="Height" default="30.0" />
    <arg name="H_mission" default="2.5" />
    <arg name="Speed" default="3.0" />

    <arg name="fcu_url" default= "/dev/ttyUSB0:921600" />
    <arg name="gcs_url" default="" />
    <arg name="tgt_system" default="1" />
    <arg name="tgt_component" default="1" />

    
    <node pkg="rohang2" type="rohang2" name="rohang2" output="screen">
    <param name="Height" value="$(arg Height)" />
    <param name="Speed" value="$(arg Speed)" />
    <param name="H_mission" value="$(arg H_mission)" />
        
    <param name="Mode" value="$(arg Mode)" />
    <!-- Set Location [LLH] -->
    <rosparam param="LLH_NoDrone"> [36.58100650, 127.52655239, 0] </rosparam>
    <rosparam param="LLH_Mission"> [36.58066411, 127.52673742, 0] </rosparam>
    <rosparam param="LLH_Return"> [36.58118480, 127.52743940, 0] </rosparam>
    <rosparam param="LLH_Ex"> [37.469445, 127.3025, 0] </rosparam>
    <!-- Set Location [ECEF] -->
    <rosparam param="ECEF_NoDrone"> [-3071638.75,4031649.68,3858899.99] </rosparam>
    <rosparam param="ECEF_Mission"> [-3071648.33,4031669.55,3858871.80] </rosparam>
    <rosparam param="ECEF_Return"> [-3071654.92,4031684.04,3858851.54] </rosparam>
    <!-- Set Location [EN] -->
    <rosparam param="EN_NoDrone"> [10, -50] </rosparam>
    <rosparam param="EN_Mission"> [0, -100] </rosparam>
    <rosparam param="EN_Return"> [0, -150] </rosparam>
    </node>

    <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
		<param name="port" value="/dev/ttyACM0"/>
		<param name="baud" value="57600"/>
    </node>

   <node pkg="rohang2" type="tlqkf" name="tlqkf">
    </node>



</launch>
